Formula/python@3.9.rb

```diff
@@ -102,6 +110,11 @@ def install
     ENV["PYTHONHOME"] = nil
     ENV["PYTHONPATH"] = nil
 
+    # Override the auto-detection in setup.py, which assumes a universal build.
+    if OS.mac?
```

Comment:
Minor nitpick:

```suggestion
    on_macos do
```

This is more consistent with what other formulae do (albeit functionally the same).

Reply:
Indeed, that looks nicer.